# Loosens the config syntax checking to allow more flexibility.
#  (Closes: #95617,#95630)

Index: logrotate-3.8.3/config.c
===================================================================
--- logrotate-3.8.3.orig/config.c	2012-10-15 14:18:29.117724327 +0100
+++ logrotate-3.8.3/config.c	2012-10-15 14:18:32.089780981 +0100
@@ -1110,7 +1110,7 @@
 
 						while (*endtag) {
 							chptr = endtag;
-							while (!isspace(*chptr) && *chptr != ',' && *chptr)
+							while (!isspace(*chptr) && /* *chptr != ',' && */ *chptr)
 								chptr++;
 
 							tabooExts = realloc(tabooExts, sizeof(*tabooExts) *
@@ -1122,8 +1122,10 @@
 							tabooCount++;
 
 							endtag = chptr;
+							/*
 							if (*endtag == ',')
-								start++;
+								endtag++;
+							*/
 							while (isspace(*endtag) && *endtag)
 								endtag++;
 						}
